fileupload.go

```diff
@@ -14,10 +13,6 @@ import (
 type FileUploadParams struct {
 	Params `form:"*"`
 
-	// File is a deprecated form of FileReader and Filename that will do the same thing, but
-	// allows referencing a file directly. Please prefer the use of FileReader and Filename instead.
-	File *os.File
-
 	// FileReader is a reader with the contents of the file that should be uploaded.
```

Comment:
Flagging that I seem to not be able to upload with `FileReader` if I don't explicitly set `Filename` too. Not sure if expected but I don't want to forget

Reply:
Hm, that shouldn't be the case, but we should look at that more closely.